### PR TITLE
Updating master branch with the notification-services branch changes

### PR DESCRIPTION
**Pull request by ashera96**

### CODE REVIEW

File: backend/notification_services/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+target
```
Comment (deshankoswatte) [inline backend/notification_services/.gitignore]: Add new line to EOF

File: backend/notification_services/src/notification_handler/main.bal [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        path: "/event",
+        methods: ["GET"]
+    }
+    resource function getEvent(http:Caller caller, http:Request request) {
+        var data = request.getJsonPayload();
+        if (data is json) {
+            io:println(data.toJsonString());
+        }
+    }
+}
```
Comment (deshankoswatte) [inline backend/notification_services/src/notification_handler/main.bal]: Add new line to EOF

File: backend/notification_services/src/nats_producer/main.bal [hunk 3]
```diff
@@ -1,13 +1,6 @@
...
 import ballerina/io;
 
-type Notification record {
-    string issueCreator?;
-    string receiver;
-    string category;
-    string description;
-};
-
 public function main() {
```
Comment (deshankoswatte) [inline backend/notification_services/src/nats_producer/main.bal:4]: Add new line after method signature.

File: backend/notification_services/src/notification_handler/main.bal [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+@http:ServiceConfig {
+    basePath: "/"
+}
+service eventListener on new http:Listener(9090) {
+
+    @http:ResourceConfig {
+        path: "/event",
+        methods: ["GET"]
+    }
+    resource function getEvent(http:Caller caller, http:Request request) {
```
Comment (deshankoswatte) [inline backend/notification_services/src/notification_handler/main.bal:20]: Add new line after method signature.

File: backend/notification_services/src/nats_producer/main.bal (hunk 3 above)
Comment (ashera96) [reply backend/notification_services/src/nats_producer/main.bal:4]: Fixed

File: backend/notification_services/src/notification_handler/main.bal (hunk 4 above)
Comment (ashera96) [reply backend/notification_services/src/notification_handler/main.bal:20]: Fixed

**Review by deshankoswatte — APPROVED** [review]

**Pull request merged**